src/main/java/seedu/address/model/company/Deadline.java

```diff
@@ -10,11 +10,9 @@
  */
 public class Deadline {
 
-
     public static final String MESSAGE_CONSTRAINTS =
-            "Please enter a valid deadline in the format YYYY-MM-DD";
-    public static final String VALIDATION_REGEX = "^(?:20[2-9][0-9]|2[1-9][0-9]{2}|19[0-9]{2})"
-            + "-(0[1-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])$";
+            "Please enter a valid deadline in the format DD-MM-YYYY";
+    public static final String VALIDATION_REGEX = "^(0[1-9]|[12][0-9]|3[01])-(0[1-9]|1[0-2])-(19[0-9]{2}|2[0-9]{3})$";
```

Comment:
Can try to consider a more comprehensive way to test if the date is valid. For single digit date input, there needs to be a 0 in front. Possible cases of bugs include leap year, month where there is no 31 days.